### PR TITLE
Fix tapas issue

### LINKED ISSUE #12060: [skipped test] to fix
https://github.com/huggingface/transformers/pull/12059 skipped failing: `tests/test_modeling_tapas.py::TapasUtilitiesTest::test_reduce_sum_vectorized`

This issue is to track its resolution so that it won't be forgotten.

@LysandreJik

### PR DESCRIPTION
**Pull request by NielsRogge**
# What does this PR do?

Fixes #12060

However, the (slow) integration tests of TAPAS that use relative position embeddings are failing for me locally, most likely due to the new version of the [torch-scatter](https://github.com/rusty1s/pytorch_scatter) dependency. I'll look into that.

Update: just tested the models in Google Colab (which has `torch 1.8.1+cu101`). Everything seems to work fine there. However, when running locally on `torch 1.8.1+cu111`, I'm getting entirely different logits/hidden states. Both are using `torch-scatter 2.7.0`.

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
Thanks for the fix!
